judge/management/commands/runmoss.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import print_function
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from moss import *
+
+from judge.models import *
+
+
+class Command(BaseCommand):
+    help = 'Checks for duplicate code using MOSS'
+
+    LANG_MAPPING = {
+        ('C++', MOSS_LANG_CC),
+        ('C', MOSS_LANG_C),
+        ('Java', MOSS_LANG_JAVA),
+        ('Python', MOSS_LANG_PYTHON),
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument('contest', help='the id of the contest')
+
+    def handle(self, *args, **options):
+        moss_api_key = getattr(settings, 'MOSS_API_KEY')
+        if moss_api_key is None:
+            print('No MOSS API Key supplied')
+            return
+        contest = options['name']
+
+        for problem in Contest.objects.get(key=contest).problems.order_by('code'):
+            print('========== %s / %s ==========' % (problem.code, problem.name))
+            for dmoj_lang, moss_lang in lang_mapping:
+                print("%s: " % dmoj_lang,end=' ')
+                subs = Submission.objects.filter(contest__participation__virtual=0, contest__participation__contest__key=contest,
```

Comment:
This looks horrible. Put a new line after `(` and indent stuff with just 4 spaces. Also, I think `-1` should be checked too, since people might solve while spectating and then go on an alt...